compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -65,6 +65,8 @@ class CompilationTests {
       compileFile("tests/pos-custom-args/i10383.scala", defaultOptions.and("-source", "future", "-deprecation", "-Xfatal-warnings")),
       compileFile("tests/pos-custom-args/i13044.scala", defaultOptions.and("-Xmax-inlines:33")),
       compileFile("tests/pos-custom-args/jdk-8-app.scala", defaultOptions.and("-release:8")),
+      compileFile("tests/pos-custom-args/single-additional-import.scala", defaultOptions.and("-Yadditional-imports:scala.annotation")),
+      compileFile("tests/pos-custom-args/multiple-additional-imports.scala", defaultOptions.and("-Yadditional-imports:scala.annotation,scala.util.matching")),
```

Comment:
Consider using a comment instead, which scales better than this mechanism.
```
grep scalac: $(find tests -name "*.scala")
```

Reply:
Sorry, I'm not following. What comment should I leave where and what should it replace?

Reply:
Hmm, I feel a little weird testing this differently than the existing import tests. Should I move those over to comments too?

Reply:
No, just the new ones.  Put then in tests/pos and add "// scalac: -Yadditional-imports:etc" at the top.

Reply:
Any reason not to move the old ones? IHMO it's weird to test them differently since they are testing such similar things.